### PR TITLE
Add tax clock embed article

### PR DESCRIPTION
**Pull request by jbothma**
taxclock-blog-wide-top
taxclock-blog-wide-mid
taxclock-blog-wide-bottom

### CODE REVIEW

File: _posts/2016-03-7-embeddable-tax-clock.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+<div class="row">
+
+</div>
+<div class="row">
+    <h2>Embed code</h2>
+    <div class="col-md-8">
+      Place this anywhere on the page where you want to embed the tax clock.
+        <pre>&lt;script type=&quot;text/javascript&quot; src=&quot;https://static.code4sa.org/taxclock/taxclock.js&quot;&gt;&lt;/script&gt;</pre>
```
Comment (longhotsummer) [inline _posts/2016-03-7-embeddable-tax-clock.md]: We don't need a div with a magic id tag any more?

File: _posts/2016-03-7-embeddable-tax-clock.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+layout: post
+title: The 2016 Tax Clock
+date: 2016-03-07
+excerpt: |
+  It’s easy to see how much of your salary is taken for taxes, but do you know how many hours are allocated to those deductions? Enter your salary to see how your tax money is being spent.
+image: https://static.code4sa.org/taxclock/preview.jpg
+author: Lenina Rassool
+---
+
+<div class="row">
```
Comment (longhotsummer) [inline _posts/2016-03-7-embeddable-tax-clock.md]: It feels like we need an intro here. "We recently launched a tool to XXX, here's how you can re-use it."

File: _posts/2016-03-7-embeddable-tax-clock.md (hunk 1 above)
Comment (jbothma) [inline _posts/2016-03-7-embeddable-tax-clock.md]: taxclock.js inserts it for you and adds the query string param to
index.html that tells the page to show the embed link.

embed.js doesn't insert it and won't show the embed link. It seemed easier
to make a new public script for embedding. I think we talked about
inserting the div if it wasn't there which is a nice feature, but it wasn't
necessary in this case unless we want to give people that flexibility. The
only benefit is that they can make our script load last while the div is
somewhere high up in their page.

Comment (longhotsummer) [inline _posts/2016-03-7-embeddable-tax-clock.md]: :+1: the simpler the better, was just checking.

File: _posts/2016-03-7-embeddable-tax-clock.md (hunk 2 above)
Comment (jbothma) [inline _posts/2016-03-7-embeddable-tax-clock.md]: What do you think of what I added?

JD

**Comment by longhotsummer** [comment]
:+1:

**Pull request merged**